server/src/test/java/io/crate/protocols/http/CrateHttpsTransportTest.java

```diff
@@ -60,6 +60,11 @@
     private static File keyStoreFile;
     private static String defaultKeyStoreType = KeyStore.getDefaultType();
 
+    @BeforeClass
+    public static void disableProcessorCheck() {
```

Comment:
This is already part of the `ESIntegTestCase`, but the test scenario here is a unit test and only extends `ESTestCase`